src/components/android-settings-container.js

```diff
@@ -74,8 +74,15 @@ class AndroidSettingsContainer extends React.Component {
             return true
         }
 
-        const androidData = this.state
-        androidData.lastUpdated = new Date().toJSON()
+        const androidData = {
```

Comment:
If this is adopting all of the properties of `state` and adding `lastUpdated`, you can use the [object rest spread operator](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Spread_syntax):

```js
const androidData = {
  ...this.state,
  lastUpdated: new Date().toJSON()
}
```

This makes a **shallow** clone of the object and adds the `lastUpdated` property - note that if any of the properties are themselves objects,  it will **not** make a copy of them.

Reply:
I see that maybe you want to exclude some properties?  In that case the way you've done it (or making `androidData` a child property of `state`) would be fine